serveradmin/__init__.py

```diff
@@ -3,3 +3,4 @@
 Copyright (c) 2018 InnoGames GmbH
 """
 
+VERSION = (1, 3, 1)
```

Comment:
I think Serveradmin and Adminapi should share the version.

Reply:
They are the same as we can only build one package via the setup.py. This makes me question whether these two things should really live in the same repository. It doesn't seem worth the hassle to separate them though, that would be a PITA.

Moving this to the __init__.py of a module we will include in the package allows us to get this information during runtime when serveradmin was installed from a package. The setup.py is not available then while this would be in the python path.

Reply:
👍  for separating the packages, but rather for somewhere in the future. ^^

Reply:
And 👍 for Gyros PITA!